server/test/repository/ProgramRepositoryTest.java

```diff
@@ -100,43 +100,13 @@ public void lookupProgram_usesCacheWhenEnabled() {
     assertThat(programCache.get(String.valueOf(two.id))).hasValue(found);
   }
 
-  @Test
-  public void loadLegacy() {
-    DB.sqlUpdate(
-            "insert into programs (name, description, block_definitions, legacy_localized_name,"
-                + " legacy_localized_description, program_type) values ('Old Schema Entry',"
-                + " 'Description', '[]', '{\"en_us\": \"name\"}', '{\"en_us\": \"description\"}',"
-                + " 'default');")
-        .execute();
-    DB.sqlUpdate(
-            "insert into versions_programs (versions_id, programs_id) values ("
-                + "(select id from versions where lifecycle_stage = 'active'),"
-                + "(select id from programs where name = 'Old Schema Entry'));")
-        .execute();
-
-    ProgramModel found =
-        versionRepo.getActiveVersion().getPrograms().stream()
-            .filter(
-                program -> program.getProgramDefinition().adminName().equals("Old Schema Entry"))
-            .findFirst()
-            .get();
-
-    assertThat(found.getProgramDefinition().adminName()).isEqualTo("Old Schema Entry");
-    assertThat(found.getProgramDefinition().adminDescription()).isEqualTo("Description");
-    assertThat(found.getProgramDefinition().localizedName())
-        .isEqualTo(LocalizedStrings.of(Locale.US, "name"));
-    assertThat(found.getProgramDefinition().localizedDescription())
-        .isEqualTo(LocalizedStrings.of(Locale.US, "description"));
-  }
-
   // Verify the StatusDefinitions default value in evolution 40 loads.
   @Test
   public void loadStatusDefinitionsEvolution() {
```

Comment:
**suggestion:** It looks like this test is failing with a null pointer exception. I think you have to add localized name and maybe localized description into the first sqlUpdate.

Reply:
@dkatzz yep that was it. The test passes for me locally.